test/testSetup.ts

```diff
@@ -158,6 +159,12 @@ export class TestSetup {
     utilities.log('');
     utilities.log(`${this.testSuiteSuffixName} - Starting authorizeDevHub()...`);
 
+    // Only need to check this once.
+    if (!TestSetup.aliasAndUserNameWereVerified) {
```

Comment:
@CristiCanizales I added a check here to make sure & validate that the alias and user name were set and match an org that's in the org list.